src/qiskit_iqm/fake_backends/chip_sample.py

```diff
@@ -0,0 +1,157 @@
+# Copyright 2022 Qiskit on IQM developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# pylint: disable=too-many-instance-attributes
+"""
+Abstract representation of an IQM chip sample.
+"""
+from dataclasses import dataclass
+from typing import Any, Union
+
+from .quantum_architectures import IQMQuantumArchitecture
+
+
+@dataclass
+class IQMChipSample:
+    """
+    Provides the specifications of a quantum chip sample.
+
+    Args:
+        quantum_architecture (IQMQuantumArchitecture): an instance of a Quantum Architecture.
```

Comment:
```suggestion
        quantum_architecture: quantum architecture of the chip
```
The types should not be repeated here, since they already are in the annotations, and the description part should ideally not mention types at all.